packages/aila/src/core/Aila.liveWithOpenAI.test.ts

```diff
@@ -0,0 +1,125 @@
+import { Aila, AilaInitializationOptions } from ".";
```

Comment:
Add a testing file to see how Aila behaves with various inputs. At the moment the tests i this file both fail, but in the UI the behaviour is correct.

@codeincontext perhaps we can chat about this to see if we can work out why the behaviour in test is different to in dev/prod? I think I'm missing something!

Reply:
Hey. Are you free tomorrow to have a look at this?